unleash-yggdrasil/src/strategy_upgrade.rs

```diff
@@ -430,7 +430,7 @@ mod tests {
             variants: None,
```

Comment:
Nothing in this file is relevant, linter got an upgrade and it's decided to fix some tests